source/patterns/@aws-solutions-constructs/aws-apigateway-dynamodb/lib/index.ts

```diff
@@ -131,10 +131,16 @@ export class ApiGatewayToDynamoDB extends Construct {
       partitionKeyName = getPartitionKeyNameFromTable(props.existingTableObj);
     }
 
+    // The helper function has been revised to return a table interface
+    // AND an optional table object instead of just a table object
+    // to match our new design guidelines. We have edited the legacy code
```

Comment:
This isn't the comment I was looking for. I was looking for:

```
// Since we are only invoking this function with an existing Table or tableProps,
// (not a table interface), we know that the implementation will always return
// a Table object and we can safely cast away the optional aspect of the type.
```